### ISSUE TITLE
index.html fails accessibility test validation

### ISSUE THREAD

**Issue by Edlme** [Edlme/github-issue-templates#7]
**Describe the bug**
The index.html file fails validation. It is not compliant with accessibility as follows:
you are currently at risk of accessibility lawsuits.

**To Reproduce**
1. Go to 'https://www.accessibilitychecker.org/'
Paste in 'https://edlme.github.io/github-issue-templates/'
Click Check website
View error

**Expected behavior**
The accessibility test should show no error on the page

**Screenshots**
Click on the following link to view the current accessibility test errors:
https://www.accessibilitychecker.org/audit/?website=https%3A%2F%2Fedlme.github.io%2Fgithub-issue-templates%2F&flag=us

**Desktop (please complete the following information):**
 - N/A

**Smartphone (please complete the following information):**
 - N/A

**Additional context**
Consider using the accessibility test to check for other errors or issues
